### ISSUE TITLE
SIDD baseline

### ISSUE THREAD

**Issue by Ryann2001** [hamadichihaoui/mash#4]
I am interested in the baseline for SIDD validation set. Could you please specify the hyper-parameters for SIDD validation baseline?
